tide_landing_page.install

```diff
@@ -1592,7 +1595,7 @@ function _get_link_value(Paragraph $paragraph) {
         $uri = $paragraph->field_paragraph_cta->uri;
         $path_details = $paragraph->get('field_paragraph_cta')->getValue();
         if (!UrlHelper::isExternal($uri)) {
-          if (str_contains($uri, 'internal:/')) {
+          if (strpos($uri, 'internal:/node/') !== FALSE) {
```

Comment:
Is this because `strpos` has better performance?

Reply:
Thanks @anthony-malkoun. It is really weird that `str_contains` should not work in our envs, as it is added in php8.0. I am thinking maybe one of other libraries defined this function.  ~~~~ I would like to change it to `strpos`.  thoughts?

Reply:
Oh, was just curious why the change. If it works then 👍 !

Reply:
Thanks!